README.md

```diff
@@ -96,6 +96,57 @@ Outputs:
     "tx_hash", // Hash of the transaction that was submitted upon successful API request.
     "execution_price" // Price that the user was charged per option contract in the order.
 }
+
```

Comment:
Can you make sure to update the `Submitting Option Order Through API` section to match the latest scripts in the example folder?